index.d.ts

```diff
@@ -7,25 +7,31 @@ export type LocaleData = {
   countries: LocalizedCountryNames
 };
 
+export type LocaleNameType = 'all' | 'official' | 'alias'
```

Comment:
Backward Compatibility with a New Option for `getName` and `getNames` Proposal:

I would like to have a new option for `getName` and `getNames` that allows the function returns `LocalizedCountryNames` type. The draft idea will be like this:

```Typescript
function getName(alpha2orAlpha3orNumeric: string | number | Alpha2Code | Alpha3Code, lang: string, type: LocaleNameType): LocalizedCountryNames
function getNames(lang: string, type: LocaleNameType): LocalizedCountryNames
```

Cases:
- `all` This always return string[]
- `official` This always return string. If the codename is an array, the functions return the first element of the array.
- `alias` This always return string. If the codename is just a string, the function return it as it is. Otherwise, it will return the second element of the codename array.